src/main/java/hudson/remoting/forward/PortForwarder.java

```diff
@@ -57,6 +57,14 @@ public PortForwarder(int localPort, Forwarder forwarder) throws IOException {
         // mark as a daemon thread by default.
         // the caller can explicitly cancel this by doing "setDaemon(false)"
         setDaemon(true);
+        setUncaughtExceptionHandler((t, e) -> {
```

Comment:
Note that `run()` below creates a nested `Port forwarding session from... ` thread. Maybe it also needs a handler